### PR TITLE
Made updates to links as they didn't seem to work

### PR DESCRIPTION
**Pull request by GinoVin**
Ready for final review - All changes have been made in >docs folder for plasma guidelines

### Proposed Changes

The plasma links below didn't seem to work. I added the full path to test if this was the issue.
Also made updates to wording, consistency and phrases.

### Potential Breaking Changes

- Ensure table in TextArea.md appears
- The Numeric Input guidelines are not appearing. On the plasma webpage seems the file names don't match (`NumericInput.md` Vs. `NumericInputConnected.md`
### Acceptance Criteria

-   [ ] The proposed changes are covered by unit tests
-   [ ] The potential breaking changes are clearly identified
-   [ ] [README.md](https://github.com/coveo/plasma/blob/master/README.md) is adjusted to reflect the proposed changes (if relevant)

### CODE REVIEW

**Comment by gdostie** [comment]
Hey thanks for the update. It makes me think that we probably have a lot more broken links (in every guidelines file). Do you think you could fix them out while you're at it?

For the deprecated autosize option, we cannot remove it yet since it is still used in the admin-ui.

**Comment by GinoVin** [comment]
> Hey thanks for the update. It makes me think that we probably have a lot more broken links (in every guidelines file). Do you think you could fix them out while you're at it?
> 
> For the deprecated autosize option, we cannot remove it yet since it is still used in the admin-ui.

💯  I'll fix all the links 👍🏼 
Sounds good for the **Autosize**

**Comment by GinoVin** [comment]
Ready for final review
@mikedidomizio
@gdostie 
@elessardCoveo
@GermainBergeron

File: packages/website/docs/Button.md [hunk 1]
```diff
@@ -34,5 +34,5 @@ When the label is hidden, add a tooltip displaying the action.
 
 If your use case doesn't match the guidelines above, consider using one of the following components instead:
 
--   [Link](#/foundations/Links) to redirect users to another section or website.
--   [Actionable items](#/form/ActionableItem) to group several less important action together.
+-   [Link](https://plasma.coveo.com/foundations/Links) - When you need to redirect users to another section or website.
```
Comment (gdostie) [inline packages/website/docs/Button.md:37]: I'm wondering if `[Link](/foundations/Links)` works, have you tried?

**Review by gdostie — APPROVED** [review]

File: packages/website/docs/NumericInput.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ Provide a descriptive title without action verbs. For example, write "Price" rat
 
 # Help Text and Instructions
 
-If the value to select must be within a range, consider indicating the minimum and maximum values, or only one of them, in a help text (e.g., "Maximum of 20").
+If the selectable value must be within a range, consider indicating the minimum and maximum values, or only one of them, in a help text (e.g., "Maximum of 20").
```
Comment (jlevesquecoveo) [inline packages/website/docs/NumericInput.md:17]: I'm not sure about this? It seems to infer that there are unselectable values?

File: packages/website/docs/RadioSelectConnected.md [hunk 3]
```diff
@@ -1,18 +1,18 @@
...
--   For a yes/no choice, such as opt in/out, use a stand-alone Checkbox](#/form/Checkbox). Example: Receive email notification.
+-   For an on/off choice, use a [Toggle](https://plasma.coveo.com/form/FlatSelect). Example: Enable dark mode.
+-   For a yes/no choice, such as opt in/out, use a stand-alone [Checkbox](https://plasma.coveo.com/form/Checkbox). Example: Receive email notification.
 -   When it’s neither, use a radio button. Examples: Choose between red or blue.
 
-Aim for seven or less options. If that's impossible, consider using the [Single select](#/form/SingleSelect) instead.
+Aim for seven or less options. If that's impossible, consider using the [Single select](https://plasma.coveo.com/form/SingleSelect) instead.
 
-Options should be sorted by most relevant unless a more suited ordering rationale applies, for example when listing size or security level options. If no rationale stands out, place options in alphanumerical order.
+Sort options by most relevant, unless a more suited ordering rationale applies. For example, when listing size or security level options. If no rationale stands out, place options in alphanumerical order.
```
Comment (jlevesquecoveo) [inline packages/website/docs/RadioSelectConnected.md:15]: Same comment as above for the "for example" sentence.

File: packages/website/docs/RadioSelectConnected.md [hunk 4]
```diff
@@ -38,14 +38,14 @@ Always preselect an option as the default. The default option can identify:
 -   The most commonly selected option when you want to help expedite the task.
     If preselecting a default option increases the risk of irreversible changes or security issues, always use the least risky option as the default.
 
-When the user needs to be able to easily revert to the default option, for instance when testing configurations, add “(recommended)” or “(default)” to the appropriate option label.
+When the user must easily revert to the default option, for instance when testing configurations, add “(recommended)” or “(default)” to the appropriate option label.
```
Comment (jlevesquecoveo) [inline packages/website/docs/RadioSelectConnected.md:41]: Again, not sure the meaning is 100% the same

File: packages/website/docs/Slider.md [hunk 5]
```diff
@@ -1,16 +1,16 @@
 # Best Practices
 
-A slider is appropriate when providing **an exact value is not important**. Sliders typically allow users to adjust the intensity of an effect such as a percentage of opacity. They are also used when an approximate value is sufficient, for example when choosing a flight departure time.
+A slider is appropriate when providing **an exact value is not important**. Sliders typically allow users to adjust the intensity of an effect, such as a percentage of opacity or when an approximate value is sufficient, for example when choosing a flight departure time. It helps visualize a range of allowed values.
```
Comment (jlevesquecoveo) [inline packages/website/docs/Slider.md:3]:
```suggestion
A slider is appropriate when providing **an exact value is not important**. Sliders typically allow users to adjust the intensity of an effect, such as a percentage of opacity, or when an approximate value is sufficient, for example when choosing a flight departure time. It helps visualize a range of allowed values. 
```

File: packages/website/docs/Slider.md [hunk 6]
```diff
@@ -1,16 +1,16 @@
...
+A slider is appropriate when providing **an exact value is not important**. Sliders typically allow users to adjust the intensity of an effect, such as a percentage of opacity or when an approximate value is sufficient, for example when choosing a flight departure time. It helps visualize a range of allowed values. 
 
-A slider visualizes the range of allowed values. Use a slider when page real estate is not an issue and when visualizing the range helps users make more informed choices.
-
-The slider should have a range of up to 100 values. If your range is very short (i.e., 5 values or less) or very large (i.e., over 50 values), consider using a [numeric input](#/form/NumericInput) instead.
+* Use a slider when page real estate is not an issue and when visualizing the range helps users make more informed choices.
+* The slider should have a range of up to 100 values. 
+* If the range is short (i.e., 5 values or less) or large (i.e., over 50 values), consider using a [numeric input](https://plasma.coveo.com/form/NumericInput) instead.
```
Comment (jlevesquecoveo) [inline packages/website/docs/Slider.md:7]: Why the bullet points?

File: packages/website/docs/Slider.md [hunk 7]
```diff
@@ -1,16 +1,16 @@
...
 
 Use sliders carefully, as they can be difficult to control on smaller devices or when the range is very large but the space is limited.
 
 ## Labeling
 
-Keep marker labels short. Aim for **only one word**.
+Keep marker labels short (i.e., a single word).
```
Comment (jlevesquecoveo) [inline packages/website/docs/Slider.md:13]:
```suggestion
Keep marker labels short (i.e., ideally a single word).
```

**Review by jlevesquecoveo — APPROVED** [review]

File: packages/website/docs/MultiSelectConnected.md [hunk 8]
```diff
@@ -1,10 +1,10 @@
 # Best Practices
 
-Use a Multi select to have users **filter and select options from a long list**. A Multi select is especially appropriate when the available list of options is very long or when space is limited.
+Use a Multi select for when users need to **filter and select options from a long list**. A Multi select is especially appropriate when the available list of options is very long or when space is limited.
 
-List options in alphanumerical order unless a more suited ordering rationale applies, for example when listing size or security level options.
+List options in alphanumerical order unless a more suited ordering rationale applies. For example, when listing size or security level options.
```
[the remark this answers is not in the document]
Comment (GinoVin) [reply packages/website/docs/MultiSelectConnected.md:5]: It makes sense to me 💭

File: packages/website/docs/NumericInput.md (hunk 2 above)
Comment (GinoVin) [inline packages/website/docs/NumericInput.md:17]:
```suggestion
If the required value must be within a range, consider indicating the minimum and maximum values, or only one of them, in a help text (e.g., "Maximum of 20").
```

File: packages/website/docs/RadioSelectConnected.md (hunk 3 above)
Comment (GinoVin) [inline packages/website/docs/RadioSelectConnected.md:15]:
```suggestion
Sort options by most relevant, unless a more suited ordering rationale applies. For example, when listing size (large to small) or security level options (lll access to limited access). If no rationale stands out, place options in alphanumerical order.
```

File: packages/website/docs/MultiSelectConnected.md (hunk 8 above)
Comment (GinoVin) [inline packages/website/docs/MultiSelectConnected.md:5]:
```suggestion
List options in alphanumerical order unless a more suited ordering rationale applies. For example, when listing size (large to small) or security level options (Full access to limited access).
```

File: packages/website/docs/RadioSelectConnected.md (hunk 4 above)
Comment (GinoVin) [inline packages/website/docs/RadioSelectConnected.md:41]:
```suggestion
When the user needs to be able to easily revert to the default option, for instance when testing configurations, add “(recommended)” or “(default)” to the appropriate option label.
```

Comment (GinoVin) [reply packages/website/docs/RadioSelectConnected.md:41]: okay, this one I'll revert

File: packages/website/docs/Slider.md (hunk 6 above)
Comment (GinoVin) [reply packages/website/docs/Slider.md:7]: I felt like it was easier to digest the use cases as a list

Comment (jlevesquecoveo) [reply packages/website/docs/Slider.md:7]: Ah, ok. To me it's just best practices like the paragraph above, that's why I don't see the point. Should we introduce them with a sentence that ends with a colon, then?

File: packages/website/docs/Slider.md [hunk 9]
```diff
@@ -1,16 +1,16 @@
 # Best Practices
 
-A slider is appropriate when providing **an exact value is not important**. Sliders typically allow users to adjust the intensity of an effect such as a percentage of opacity. They are also used when an approximate value is sufficient, for example when choosing a flight departure time.
+A slider is appropriate when providing **an exact value is not important**. Sliders typically allow users to adjust the intensity of an effect, such as a percentage of opacity, or when an approximate value is sufficient, for example when choosing a flight departure time. It helps visualize a range of allowed values.
```
Comment (GinoVin) [inline packages/website/docs/Slider.md:3]:
```suggestion
A slider is appropriate when providing **an exact value is not important**. Sliders typically allow users to adjust the intensity of an effect, such as a percentage of opacity, or when an approximate value is sufficient, for example when choosing a flight departure time. It helps visualize a range of allowed values. Consider the following:
```

**Comment by GinoVin** [comment]
@gdostie Hi Gaël, I'm unable to merge (because I don't have the auth).
The changes are ready to merge, unless you wanted to have anyone else from your team take a look at the suggested changes :)

**Pull request merged**